### ISSUE TITLE
It shows no preview information

### ISSUE THREAD

**Issue by zhangjieluke** [dcartertod/zotero-plugins#17]
The preview always switches to anthoer display mode, like the right graph. Only reinstalling the zotero preview plugin can make it return the normal mode, like the one in the left graph.
Is there way to solve it?
![Picture1](https://user-images.githubusercontent.com/66522779/168965935-74062d68-6c26-4265-9dce-45c30e105b10.png)

**Comment by dcartertod** [comment]
What version of Zotero are you running?

**Comment by dcartertod** [comment]
If you go to Edit -> Preferences -> Style Editor, what do you see?

**Comment by zhangjieluke** [comment]
This is what I see from  Edit -> Preferences -> Style Editor, when it is working well.

![Capture](https://user-images.githubusercontent.com/66522779/169049570-119c740e-4722-4c5b-8ac2-8fb262322184.PNG)
)

**Comment by dcartertod** [comment]
Did you choose a preview style?

Does Tools->Developer->Error console have any errors specific to zoteropreview?

**Comment by zhangjieluke** [comment]
I did choose a preview style. I switched between several styles.

I just filter all messages with 'preview' or 'zoteropreview' in the Error console. It shows nothing, which means there is no error related to zoteropreview.

When that problem appears next time, I will check the error console again, and will post the results here.

**Comment by drewx0r** [comment]
I downgraded due to this same issue. Maybe I'll try upgrading again and seeing if the problem persists with 6.0.8.

**Comment by dcartertod** [comment]
I can't reproduce it. Any debug or error logs would be helpful.

**Comment by drewx0r** [comment]
It's pretty much the same as zhangjieluke's. The style editor shows the bibliography, but the preview pane doesn't. No errors for zoteropreview in the error log. 

I disabled all other addons and it worked when I did. I'm going to do some digging and try to figure out what it's conflicting with.

**Comment by drewx0r** [comment]
OK, I disabled all addons, then (eventually, after trying to track down a single addon causing the conflict), i reenabled all the plugins, and it works. So I don't know what the deal was, but the disable/reenable seems to have either cleared out some weird cache issue or washed away whatever gremlins were causing this error. Very odd, but it looks good now. Thanks for making this. It's really nice to have.

**Comment by dcartertod** [comment]
Thank you

**Comment by zhangjieluke** [comment]
I may find a solution to this issue. Today when I went to Edit -> Preferences -> Export, the **item format** is loading. Then I chose one style. I realized that the Bibliography started to show the information, like this.
![image](https://user-images.githubusercontent.com/66522779/169662322-3aca0cc9-665f-435f-b0fb-cae608f9b880.png)

This is inspired by the setting for [mgmeyers's

obsidian-zotero-desktop-connector](https://github.com/mgmeyers/obsidian-zotero-desktop-connector)

**Comment by dcartertod** [comment]
Yeah, you have to have an export format set, but I thought I'd accounted for one not being. If someone gets this issue, I'd appreciate them running the following code under Tools->Developer->Run Javascript

var qc = Zotero.QuickCopy;
var format = qc.getFormatFromURL(qc.lastActiveURL);
format = Zotero.QuickCopy.unserializeSetting(format);

**Comment by xiaweichuliang** [comment]
> I may find a solution to this issue. Today when I went to Edit -> Preferences -> Export, the **item format** is loading. Then I chose one style. I realized that the Bibliography started to show the information, like this. ![image](https://user-images.githubusercontent.com/66522779/169662322-3aca0cc9-665f-435f-b0fb-cae608f9b880.png)
> 
> This is inspired by the setting for [mgmeyers's
> 
> obsidian-zotero-desktop-connector](https://github.com/mgmeyers/obsidian-zotero-desktop-connector)

Give you a thumb!

**Comment by alekhe** [comment]
> Yeah, you have to have an export format set, but I thought I'd accounted for one not being. If someone gets this issue, I'd appreciate them running the following code under Tools->Developer->Run Javascript

Is it possible to have a different export format? I use "Better BibTeX Quick Copy". However it would be still nice to have a preview pane.

> var qc = Zotero.QuickCopy;
> var format = qc.getFormatFromURL(qc.lastActiveURL);
> format = Zotero.QuickCopy.unserializeSetting(format);

```
{
    "mode": "export"
    "contentType": ""
    "id": "a515a220-6fef-45ea-9842-8025dfebcc8f"
    "locale": ""
}
```

**Comment by dcartertod** [comment]
If you the "Choose Preview Style" button, it should set a preview style just for the preview mode.

**Comment by alekhe** [comment]
Just (re?)installed v0.0.8 from the “Releases” and it works fine.

However, I don’t know if that was an “outdated-version” issue, since I installed this plugin last month and the previous version v0.0.7 is dated Sep 2021. The strange thing also is that in Zotero on the plugin info page it show “ZoteroPreview 0.1.0” and the previously installed version was shown as “0.1.2”.

**Comment by alekhe** [comment]
Unfortunately, the version I installed from https://github.com/dcartertod/zotero-plugins/releases/tag/0.0.8, got an update and the updated version does not work.

As a side note, again. The version from https://github.com/dcartertod/zotero-plugins/releases/tag/0.0.8 was shown as 0.1.0 in the Zotero extensions page. After the update, the version is 0.1.2 (which does not work).

**Comment by dcartertod** [comment]
I'll see what I can do.

**Comment by alekhe** [comment]
@drewx0r,
It did work. Thanks for the hack. Interesting behavior.

**Comment by dcartertod** [comment]
Just released an update 0.1.3 - should be available via Add-ons -> Check for updates

**Comment by alekhe** [comment]
@dcartertod,
It works. Thank you!
